deployments/admission-controllers/schedulername-mutation/admission_util.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright 2019 Cloudera, Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+basedir="$(dirname "$0")"
+SERVICE=`cat configs.properties | grep service | cut -d "=" -f 2`
+SECRET=`cat configs.properties | grep secret | cut -d "=" -f 2`
+NAMESPACE=`cat configs.properties | grep namespace | cut -d "=" -f 2`
+
+delete_resources() {
+  kubectl delete -f server.yaml
+  kubectl delete -n ${NAMESPACE} secret ${SECRET}
+  kubectl delete -n ${NAMESPACE} certificatesigningrequest.certificates.k8s.io ${SERVICE}.${NAMESPACE}
+  rm -rf server.yaml
```

Comment:
Let's not delete namespace here, in case the user specifies an existing namespace for the admission controller. I think it might be running with yunikorn-scheduler within the same place. So I added it back after a second thought.

Reply:
OK, that makes sense